content/Software and Downloads/IDE Settings/Run-arduino-debug-exe-Windows.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Run arduino_debug.exe (Windows)"
+---
+
+The Java-based version of Arduino IDE (1.x) for Windows includes a debug tool for troubleshooting.
```

Comment:
```suggestion
Arduino IDE 1.x for Windows includes an alternative executable that provides log output to use for troubleshooting problems with the IDE.
```

- The user does not care whatsoever which language the Arduino IDE happens to be written in. That information is completely irrelevant to the content of the tutorial
- The term "debug tool" can be confusing because it might be interpreted as meaning a tool for debugging the sketch code (such as the debugger tool Arduino IDE 2.x has)
- Make it clear that this is for troubleshooting problems with the IDE.